hibernate-core/src/test/java/org/hibernate/orm/test/schemamanager/SchemaManagerExplicitSchemaTest.java

```diff
@@ -65,6 +66,13 @@ private Long countBooks(SessionImplementor s) {
 		catch (SchemaManagementException e) {
 			assertTrue( e.getMessage().contains("ForTesting") );
 		}
+		try {
+			factory.getSchemaManager().validate();
+			fail();
+		}
+		catch (SchemaValidationException e) {
+			assertTrue( e.getMessage().contains("ForTesting") );
+		}
```

Comment:
It's better to use assertj `assertThatExceptionOfType`
```java
assertThatExceptionOfType(SchemaValidationException.class)
  .isThrownBy(() -> {
     factory.getSchemaManager().validate();
}).hasMessageContaining("ForTesting");
```

Reply:
OK, cool, thanks, I'll keep that in mind for next time.